manual/src/cmds/intf-c.etex

```diff
@@ -414,7 +414,7 @@ bytecode executable (so that the bytecode from "unix.cma" and
 \section{s:c-value}{The \texttt{value} type}
 
 All OCaml objects are represented by the C type "value",
-defined in the include file "caml/mlvalues.h", along with macros to
+defined in the include file "<caml/mlvalues.h>", along with macros to
```

Comment:
Have you tested that these render correctly in the LaTeX version?
(I just recall problems with the rendering of '<' and '>' in the past) :thinking:

Reply:
OK, thanks for checking! :pray: